source/qdk_package/src/qdk/qiskit.py

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""Re-export shim for the optional qiskit package as qdk.qiskit.
+
+If qiskit is not installed (with the qdk[qiskit] extra), importing this
+module raises an ImportError describing how to enable it.
+"""
+
+from __future__ import annotations
+
+try:
+    from qiskit import *
```

Comment:
We shouldn't reexport everything from qiskit under the qdk. They should import from qiskit for the things they need from qiskit.